jinja2/filters.py

```diff
@@ -952,8 +952,6 @@ def _select_or_reject(args, kwargs, modfunc, lookup_attr):
     'reverse':              do_reverse,
     'center':               do_center,
     'indent':               do_indent,
-    'title':                do_title,
-    'capitalize':           do_capitalize,
```

Comment:
how about just taking this PR as is and then doing minor additional fixes on your own?